index.html

```diff
@@ -0,0 +1,322 @@
+      href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="toolbar">
+      <nav class="nav-bar">
+        <a href="#"><img src="./images/logo.png" alt="logo" /></a>
+        <ul class="sid-cons">
+          <li class="p">Portoflio</li>
+          <li class="a">About</li>
+          <li class="c">Contact</li>
+          <li class="material-symbols-outlined">menu</li>
+        </ul>
+      </nav>
+    <div class="headline-container">
+      <div class="items1">
+        <h1 class="items item1">I'm Gasangwa Glad to see you !</h1>
+        <p class="items item3">
+          I am a software developer. I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need to be coded, don't hesitate to
+          contact me.
+        </p>
+        <div>
+          <h4 class="contact">LET'S CONNECT</h4>
+          <ul class="icons">
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/twitter.png"
+                  alt="twitter-icon"
+                  class="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/linkedin.png"
+                  alt="linkedin-icon"
+                  class="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/medium.png"
+                  alt="medium-icon"
+                  class="medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/github.png"
+                  alt="github-icon"
+                  class="github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/angellist.png"
+                  alt="angellist-icon"
+                  class="angellist"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    </div>
+    <!--html of the middle section-->
+    <section class="middle-section">
+      <div class="card1">
+        <div class="snapshoot">
+          <img src="./images/screenshot1.png" alt="screenshot" class="responsive"/>
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Thomas</h2>
+            <ul class="canop">
+              <li>CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
```

Comment:
- [ ] Please make sure the project cards are aligned in the center.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/166200523-65575f65-c9d2-47c3-aaf1-2ee2d80fadc1.png)